metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -195,18 +198,18 @@ def extract_lineage(
             )
             upstreams.append(upstream_table_class)
 
-            if len(upstreams) > 0:
-                upstream_lineage = UpstreamLineageClass(upstreams=upstreams)
-                logger.debug(
-                    f"DataSet Lineage = {ds_urn} and its lineage = {upstream_lineage}"
-                )
-                mcp = MetadataChangeProposalWrapper(
-                    entityType=Constant.DATASET,
-                    changeType=ChangeTypeClass.UPSERT,
-                    entityUrn=ds_urn,
-                    aspect=upstream_lineage,
-                )
-                mcps.append(mcp)
+        if len(upstreams) > 0:
```

Comment:
is this unindent intentional? was this just a bug?

Reply:
it was intentional, fixing a subtle bug where if there were two upstreams then we'd emit two upstreamLineage mcps instead of a single one

Reply:
oh wow. thanks!